src/DatePicker/index.js

```diff
@@ -1,27 +1,17 @@
 import delegate from 'delegate'
 import defaults from './lib/defaults'
-import zeroPad from './lib/zeroPad'
+import zeroPad  from './lib/zeroPad'
+import keys     from './lib/keys'
 
 export default class DatePicker {
   constructor(el, options = {}) {
-    this.constants = Object.assign({}, defaults, options)
+    this.props = Object.assign({}, defaults, options)
```

Comment:
This isn't supported at all in IE, from what I'm reading. Do you have a polyfill?